x-pack/legacy/plugins/security/public/views/management/edit_role/index.js

```diff
@@ -102,10 +102,16 @@ routes.when(`${EDIT_ROLES_PATH}/:name?`, {
     }
   },
   controllerAs: 'editRole',
-  controller($injector, $scope, $http, enableSpaceAwarePrivileges) {
+  controller($injector, $scope, $http, enableSpaceAwarePrivileges, kbnUrl) {
     const $route = $injector.get('$route');
     const Private = $injector.get('Private');
 
+    const action = $route.current.params.action;
+
+    if (!['edit', 'clone'].includes(action)) {
```

Comment:
Instead of doing this and what we're doing on line 32, could we create a variable which is the route definition and do:

```
const routeDefinition = { ... };
routes.when(`${EDIT_ROLES_PATH}/:name`, routeDefinition);
routes.when(`${CLONE_ROLES_PATH}/:name`, routeDefinition);
```